### PR TITLE
[Components with more than 1 service] LPS-179348 Refactor ScreenNavigationCategory components from Commerce modules part 2

### PR DESCRIPTION
**Pull request by daiazius**
Related pull: https://github.com/liferay-core-infra/liferay-portal/pull/3987
Issue: https://issues.liferay.com/browse/LPS-179348

### CODE REVIEW

**Comment by daiazius** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
@daiazius No regressions found. There is a conflict in /commerce-order-web/bnd.bnd. Please resolve it.

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:relevant

**Comment by daiazius** [comment]
ci:test:startup

**Comment by zhangyan0701** [comment]
No regressions found.

File: modules/apps/commerce/commerce-payment-api/src/main/java/com/liferay/commerce/payment/constants/CommercePaymentScreenNavigationConstants.java [hunk 1]
```diff
@@ -26,15 +26,19 @@ public class CommercePaymentScreenNavigationConstants {
 		"details";
 
 	public static final String
-		CATEGORY_KEY_COMMERCE_PAYMENT_METHOD_RESTRICTIONS = "restrictions";
+		CATEGORY_KEY_COMMERCE_PAYMENT_METHOD_RESTRICTIONS =
+			"payment-restrictions";
 
 	public static final String CATEGORY_KEY_QUALIFIERS = "qualifiers";
 
 	public static final String ENTRY_KEY_COMMERCE_PAYMENT_METHOD_DETAILS =
```
Comment (dantewang) [inline modules/apps/commerce/commerce-payment-api/src/main/java/com/liferay/commerce/payment/constants/CommercePaymentScreenNavigationConstants.java:34]: @daiazius Please remove these 2 entry keys and delegate usage to getCategoryKey() to be consistent with other classes in this whole PR.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by daiazius** [comment]
ci:test:sf

**Comment by daiazius** [comment]
ci:test:startup

**Comment by daiazius** [comment]
ci:test:commerce

**Comment by dantewang** [comment]
See https://github.com/liferay-core-infra/liferay-portal/pull/4282

**Pull request closed without merging**